packages/api/graph/schema.graphql

```diff
@@ -121,13 +121,55 @@ type Hustler implements Node {
   accessory: Item
 }
 
+type PaymentToken implements Node {
+  id: ID!
+  address: String
+  type: String
+  symbol: String
+  price: Float
+}
+
+type Listing implements Node {
+  id: ID!
+  source: Source
+  active: Boolean!
+  inputs: [Asset]!
+  outputs: [Asset]!
+}
+
+enum AssetType {
+  ETH
+  EQUIPMENT
+  HUSTLER
+  TURF
+  PAPER
+}
+
+enum Source {
+  OPENSEA
+  SWAPMEET
+}
+
+type Asset implements Node {
+  id: ID!
+  address: Address
+  type: AssetType
+  symbol: String
+  amount: BigInt
+  assetId: BigInt
+  price: Float
```

Comment:
is price here USD price of the underlying asset? imo we shouldn't expose that (hard to compute for most assets) and can prob remove `priceType` too